week2/homework/src/read.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const fs = require('fs');
+
+exports.reading = function() {
```

Comment:
I would keep `read` and `write` functionalities in a one file called `utiles` instead of multiple.

what if you have dozens of features would you keep them in a many files, other than what you necessarily have?

Reply:
One file called `utiles` , this makes things much better. 
And I agree about descriptive naming issue. 
Thank you for your time on reviewing this homework.